clientv3/balancer.go

```diff
@@ -71,8 +80,15 @@ func newSimpleBalancer(eps []string) *simpleBalancer {
 		notifyCh: notifyCh,
 		readyc:   make(chan struct{}),
 		upc:      make(chan struct{}),
+		stopc:    make(chan struct{}),
+		donec:    make(chan struct{}),
+		downc:    make(chan struct{}),
 		host2ep:  getHost2ep(eps),
 	}
+	// downc must be non-nil to avoid closing on a nil chan
```

Comment:
don't need to worry about this any more if `stopc` controls the notify goroutine?